### ISSUE TITLE
Carestream Trophy Panoramic images

### ISSUE THREAD

**Issue by leonardorame** [horosproject/horos#716]
Hi, I'm looking to solve an issue that shows on almost all OpenSource viewers (Weasis, CornerstoneJs and DCMTK library), but in Horos is fixed!.

The issue is related to images generated by Carestream Trophy dental equipments, on Panoramic images. They show too bright.

You can take a look at my analysis here: https://github.com/nroduit/Weasis/discussions/335

Can you help me with this?, or better yet, point me to the source code that applies the correct VOI (or algorithm) to correctly display these images.

Regards,
Leonardo.

**Comment by aglv** [comment]
any chance you could send me one of these files? I could try and figure out how Horos decides to use the 127/255 wl/ww, and how it gets to display it the way we assume is correct...
Please send me one of these DICOM files at breaks-07vibrant@icloud.com

**Comment by leonardorame** [comment]
Hi @aglv, I've sent a sample image to your email, please confirm receipt.

**Comment by aglv** [comment]
@leonardorame In the Weasis forum you say Horos displays the image exactly like the acquisition software. 
I was only able to get an image close to that by applying the Opacity: Smooth Table adjustment. Is this how you got to show it correctly? Otherwise, without adjustments, Horos shows it like the other screenshots you put.

**Comment by leonardorame** [comment]
@aglv, you are right, the issue seems to be more complex than I originally thought. I got confused because the samples I received where taken in the same site, but using two different modalities.

Today I'll open the same study with the Carestream application and bring back the results.

**Comment by nroduit** [comment]
I found orthopantomograms produced by a Carestream modality and they have Window/Level values in the standard DICOM attributes.

The best approach to solve this problem is to modify the acquisition configuration to have Window/Level values or a VOI LUT in the standard DICOM attributes. It is up to the vendor to produce a correct file and not to the DICOM viewers to adapt to all the proprietary information (this is the primary objective of DICOM). Otherwise, the windowing has to be adjusted in the viewer.
